src/Query/Query.php

```diff
@@ -627,7 +589,7 @@ public function addSelect($columns): self
      *
      * @return array
      */
-    protected function normalizeSelect($columns): array
+    protected function normalizeSelect(array|ExpressionInterface|string $columns): array
```

Comment:
array|string|ExpressionInterface vs array|ExpressionInterface|string
May be order by simple types (a-z), classes&interfaces (a-z), null?
And add this to codestyle?

Reply:
The order of the classes is fixed automatically by style.ci.